src/envoy/http/mixer/filter.cc

```diff
@@ -58,7 +58,7 @@ void Filter::ReadPerRouteConfig(
 
 FilterHeadersStatus Filter::decodeHeaders(HeaderMap& headers, bool) {
   ENVOY_LOG(debug, "Called Mixer::Filter : {}", __func__);
-  request_total_size_ += headers.byteSize();
+  request_total_size_ += headers.byteSize().value_or(0);
```

Comment:
I think this should be `headers.refreshByteSize()` (see: #2447). cc @lambdai

Reply:
Done.